src/types.jl

```diff
@@ -241,9 +241,9 @@ struct Camera
     resolution::Observable{Vec2f}
 
     """
-    Focal point of the camera, used for e.g. camera synchronized light direction.
+    Direction in which the camera looks.
     """
-    lookat::Observable{Vec3f}
```

Comment:
I added this in the lighting pr but didn't end up using it. Swapped it out for view_direction here, which is used to sort planes. Might also be useful in volume rendering?